### ISSUE TITLE
[Feature request] Allow binding the same Dreamcast controls to different inputs

### ISSUE THREAD

**Issue by dezraj** [flyinghead/flycast#1533]
**Is your feature request related to a problem? Please describe.**
In Alone in the Dark 4 we control the character movement with the directional pad and the flashlight movement with the analog stick. I think it's more comfortable to control the character with the analog stick, so I changed these controls in the inputs settings so I could control the character with the left analog stick and the flashlight with the right analog stick in my controller. Now I have the directional pad free in the controller and would also like to also bind it to the character movement. In some parts of the game using the d-pad lets the control more comfortable. Unfortunately the program does not allow mapping the same Dreamcast controller control to a different input in my controller.

**Describe the solution you'd like**
The program could allow us to bind the same Dreamcast controller input to a different input in the configured controller. Most controllers nowadays have two analog sticks and one d-pad, and some have more buttons than what the DC controller had, so it's possible to duplicate the Dreamcast controls at some extent. This would be useful for some games such as Alone in the Dark 4.

**Additional context**
We currently have "Map" and "Unmap" options in the input settings. Maybe we could have an "Add" option to allow adding a new input source to the same control, or the "Map" option could simply allow registering more than one input to the same Dreamcast control. And then "Unmap" could be renamed to "Clear" because it would clear all the inputs associated to that Dreamcast control.

Thank you so much in advance for considering this request.

**Comment by CO-Windler** [comment]
Yes this would make sense. Also arcade driving games often have complex controls (e.g. F355 Challenge has paddle shift in medium mode but stick shift in simulator mode). Assigning multiple controls to the same input and vice versa like in MAME would be useful.
